katello.dot

```diff
@@ -2,108 +2,120 @@
 
 [Browser] as browser
 
-node "server" {
-  [Postgres] as postgres
-  [Candlepin] as candlepin
+node "Main Environment" {
+	
+	node "Server" {
+	  [Postgres] as postgres
+	  
 
-  [Elasticsearch] as elasticsearch
-  [MongoDB] as mongo_server
-  [Qpid] as qpidd_server
-  [Dispatch Router] as qdrouterd_server
-  [Smart Proxy] as server_smart_proxy
+	  [MongoDB] as mongo_server
+	  [Qpid] as qpidd_server
+	  [Dispatch Router] as qdrouterd_server
+	  [Smart Proxy] as server_smart_proxy
+	  
+	  node "Tomcat" as tomcat_server {
+		  [Candlepin] as candlepin
+	  }
 
-  node "Apache" as server_apache {
+	  node "Apache" as server_apache {
 
-    node "Pulp" as pulp_server {
-      [API] as pulp_server_api
-      [pulp_* processes] as pulp_server_backend
-    }
+		node "Pulp" as pulp_server {
+		  [API] as pulp_server_api
+		  [pulp_* processes] as pulp_server_backend
```

Comment:
Care to take a stab at moving these out of the Apache node? Given they are actually celery processes that run independent of Apache that would clarify some aspects of it.

Reply:
yeap, totally agree. This process internally on the apache is related to mod_wsgi .., working on the new version.